packages/ui-extensions-server-kit/src/components/ActionSet/ActionSpacer/ActionSpacer.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import React, {useState} from 'react';
+import {ComponentStory, ComponentMeta} from '@storybook/react';
+
+import {Action} from '../Action';
+import {RefreshMinor, HideMinor} from '../../Icon';
+import styles from '../../../../.storybook/styles.module.scss';
+import {ActionSet} from '../ActionSet';
+import {mockExtensions} from '../../../testing';
+
+import {ActionSpacer as ActionSpacerComponent} from './ActionSpacer';
+
+// More on default export: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
+export default {
+  title: 'Components/ActionSpacer',
+  component: ActionSpacerComponent,
+} as ComponentMeta<typeof ActionSpacerComponent>;
+
+// More on component templates: https://storybook.js.org/docs/react/writing-stories/introduction#using-args
+const Template: ComponentStory<any> = () => {
+  const [hasSpacer, setSpacer] = useState(false);
+  const insertActionSpacer = () => setSpacer((state) => !state);
+  const extensions = mockExtensions();
+  return (
+    <>
+      <ActionSet extensions={extensions}>
```

Comment:
Can you wrap this ActionSet in a parent so that it doesn't float right. It's hard to see, so better have it on the left.

Reply:
this way it is consistent across the storybook